modules/Bio/EnsEMBL/Compara/PipeConfig/Plants/LoadMembers_conf.pm

```diff
@@ -50,7 +50,7 @@ sub default_options {
 
     # names of species we don't want to reuse this time
         #'do_not_reuse_list' => [ 'oryza_indica', 'vitis_vinifera' ],
-        'do_not_reuse_list' => [ ],
+        'do_not_reuse_list' => [ 'physcomitrium_patens' ],
```

Comment:
As we have discussed, this is a change only affecting e104 (and not required in e105+), so this shouldn't be merged.